src/selection.js

```diff
@@ -333,6 +333,8 @@
         },
 
         cancel: function() {
+            this.outerTracker.setTracking(false);
```

Comment:
Might be good to add a comment for why this is here, since it's not obvious just by looking at the code.

Reply:
Fair point, will do tomorrow as I am busy today